packages/core/docs/commercetools/changelog/5419.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  description: 'New part of checkout - shipping details, inside core and commercetools',
+  link: 'https://github.com/vuestorefront/vue-storefront/pull/5552',
+  isBreaking: true,
+  breakingChanges: [
+    {
+      module: 'commercetools',
```

Comment:
why removing getters is commercetools-specific BC? it will affect the integrators

Reply:
Richtig